contrib/make_hdf.py

```diff
@@ -1,7 +1,9 @@
 #!/usr/bin/env python
 
+import cPickle
```

Comment:
Probably this should be just pickle

Reply:
This `contrib/make_hdf.py` is a Python2 file with `print` statement, `.iter*()` dict methods and `raw_input()`. I can rewrite it to Python3, but I am wondering nobody has touched this file since 2012.

Reply:
Yes please, keeping Python2 does not make sense.
In general bench and contrib have never received too much love :/

Reply:
That was not only Python 3 migration. That was several versions of PyTables worth updating. There are no tests, but the script `make_hdf.py` does something now. Please see the second commit.

Reply:
thanks @eumiro